src/NotificationTargetCommonITILObject.php

```diff
@@ -1731,6 +1857,14 @@ public function getTags()
             $objettype . '.openbyuser'            => __('Writer'),
             $objettype . '.lastupdater'           => __('Last updater'),
             $objettype . '.assigntousers'         => __('Assigned to technicians'),
+            'assigntousers.phone'                 => Phone::getTypeName(1),
+            'assigntousers.phone2'                => __('Phone 2'),
+            'assigntousers.mobile'                => __('Mobile phone'),
+            'assigntousers.category'              => _n('Category', 'Categories', 1),
+            'assigntousers.location'              => __('User location'),
+            'assigntousers.address'               => __('Address'),
+            'assigntousers.postcode'              => __('Postal code'),
+            'assigntousers.town'                  => __('City'),
```

Comment:
```suggestion
            'actors.phone'                 => Phone::getTypeName(1),
            'actors.phone2'                => __('Phone 2'),
            'actors.mobile'                => __('Mobile phone'),
            'actors.category'              => _n('Category', 'Categories', 1),
            'actors.location'              => __('Actor location'),
            'actors.address'               => __('Address'),
            'actors.postcode'              => __('Postal code'),
            'actors.town'                  => __('City'),
```

You should also add `itemtype`, `actortype`, `id`, `name`, `title`, `email`, ...